charts/kyverno-policies/templates/_helpers.tpl

```diff
@@ -56,6 +56,17 @@ helm.sh/chart: {{ template "kyverno-policies.chart" . }}
 {{- end -}}
 {{- end -}}
 
+{{/* Set if custom policies are managed */}}
+{{ include "kyverno-policies.customPolicies" ( dict "value" .Values.path.to.the.Value "context" $) }}
+*/}}
```

Comment:
This `include` is very confusing and appears to be documentation but I see uneven number of `/*` and `*/`. I'd recommend just follow the pattern with other defined functions and just have a single line description:

```suggestion
{{/* Set if custom policies are managed */}}
```